tests/unit/botocore/test_handlers.py

```diff
@@ -484,6 +485,9 @@ def test_route53_resource_id(self):
                         'DelegationSetId': {
                             'shape': 'DelegationSetId'
                         },
+                        'DelegationSetId': {
+                            'shape': 'ChangeId'
+                        },
```

Comment:
I'm not sure if this should be part of the diff based on the commit: https://github.com/boto/botocore/commit/6c862b545ea77d8ee94267cf768ef3643aee8450